docs/core/extensions/cloud-service.md

```diff
@@ -279,15 +279,15 @@ Provide the appropriate values:
 - `<instance name>`: the name of the container instance.
 - `<registry name>`: the name of the container registry.
 - `<image name>`: the name of the image.
-- `<password>`: the password to the container registry &mdash; you can get this from the Azure Portal, Container Registry resource > **Access Keys**.
+- `<password>`: the password to the container registry—you can get this from the Azure portal, Container Registry resource > **Access Keys**.
 
 :::zone-end
 :::zone target="docs" pivot="visualstudio"
 
 To create a container instance, you'll need to [create a new resource](https://ms.portal.azure.com/#create/Microsoft.ContainerInstances) in the Azure portal.
 
 1. Select the same **Subscription**, and corresponding **Resource group** from the previous section.
-1. Enter a **Container name** &mdash; `appcloudservice-container`.
+1. Enter a **Container name**&mdash;`appcloudservice-container`.
```

Comment:
Just curious why you eliminated `&mdash` elsewhere but left it here.

Reply:
I used the suggested Acrolinx text elsewhere, which included replacing the encoded mdash. I think in this case, Acrolinx didn't know how to replace it because of the backtick that came afterwards.